packit_service/constants.py

```diff
@@ -246,11 +246,23 @@ def from_number(number: int):
     "Not allowed to build in {copr_project} Copr project."
 )
 CUSTOM_COPR_PROJECT_NOT_ALLOWED_CONTENT = (
-    "Your git-forge project is not allowed to use the configured `{copr_project}` Copr project.\n\n"
-    f"Please, [let us know]({CONTACTS_URL}) "
-    "if you need this git project to be allowed. "
-    "We are working with the Copr team on a way "
-    "how to make this easily configurable in the Copr web interface.\n"
+    "Your git-forge project is not allowed to use "
+    "the configured `{copr_project}` Copr project.\n\n"
+    "Please, add this git-forge project `{forge_project}` "
+    "to `Packit allowed forge projects`"
+    "in the [Copr project settings](https://copr.fedorainfracloud.org/coprs/{copr_project}/edit/#packit_forge_projects_allowed). "  # noqa
+)
+
+CUSTOM_COPR_PROJECT_ALLOWED_IN_PACKIT_CONFIG = (
+    "Permissions to build git-forge projects in Copr "
+    "should now be granted in Copr server.\n"
+    "Your git-forge project `{forge_project}` has permissions "
+    "to build in `{copr_project}` Copr project, "
+    "but they are granted by Packit, "
+    "this permissions will be removed soon.\n"
+    "Please, add this git-forge project `{forge_project}` "
```

Comment:
```suggestion
    "Your git-forge project `{forge_project}` has permissions "
    "to build in `{copr_project}` Copr project configured in Packit. "
    "However, we migrated to the solution where you can configure"
    "the allowed git-forge projects in Copr yourself and will remove the configuration"
    "in Packit for the allowed projects soon. Therefore, please, add this git-forge project `{forge_project}` "
```

it's hard to write this so that it is not too long and clear, would this maybe be more clear?

Reply:
I can't judge english :) so let try with your sentences.